lib/SimplePie/SimplePie/Item.php

```diff
@@ -427,7 +427,16 @@ public function get_thumbnail()
 		{
 			if ($return = $this->get_item_tags(SIMPLEPIE_NAMESPACE_MEDIARSS, 'thumbnail'))
 			{
-				$this->data['thumbnail'] = $return[0]['attribs'][''];
+				$thumbnail = $return[0]['attribs'][''];
+				if (empty($thumbnail['url']))
+				{
+					$this->data['thumbnail'] = null;
+				}
+				else
+				{
+					$thumbnail['url'] = $this->sanitize($thumbnail['url'], SIMPLEPIE_CONSTRUCT_IRI);
```

Comment:
The thumbnail URL was previously returned unsanitized and unescaped, as opposed to other URLs returned by SimplePie

Reply:
TODO: provide a `base` as parameter for `absolutize_url` to work. Check that other enclosure URLs are correct in this regard.

Reply:
Upstream PR: https://github.com/simplepie/simplepie/pull/770